app/notifications/notifications.component.css

```diff
@@ -0,0 +1,7 @@
+.filter-options {
+  color: #00205B;
+}
+
+.select-layout {
+  width: 250px;
```

Comment:
ah, ok. But only in FF ;)
I tested in Chrome and it was fine there. thanks for clarification.